src/app/ClusterStateCache.cpp

```diff
@@ -16,9 +16,9 @@
  *    limitations under the License.
  */
 
-#include "system/SystemPacketBuffer.h"
 #include <app/ClusterStateCache.h>
```

Comment:
I would recommend to split the include file related with the cpp file from the rest of the includes, so it will be guaranteed that this include is loaded as a first one. Such ordering validates the include file that it's self contained. In this particular case it's not visible, but since we've got includes autosort, one can add another include to that file which can appear before the `<app/ClusterStateCache.h>`.